clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureComputeClient.groovy

```diff
@@ -161,8 +164,7 @@ public class AzureComputeClient extends AzureBaseClient {
   }
 
   ServiceResponse<Void> destroyServerGroup(String resourceGroupName, String serverGroupName) {
-    VirtualMachineScaleSetsOperations ops = getAzureOps(
-      client.&getVirtualMachineScaleSetsOperations, "Get operations object", "Failed to get operation object") as VirtualMachineScaleSetsOperations
+    VirtualMachineScaleSetsOperations ops = client.getVirtualMachineScaleSetsOperations()
```

Comment:
Yes. This method instantiates the class that declares the "operations" (APIs) that are available. It also does initialization of the retrofit client (which still doesn't make a rest call). Unfortunately, per the SDK team, the initialization of the retrofit client can be expensive and they have recommended that we limit our calls to instantiate the "operations" classes when possible.